code/src/test_srcs/test__sbi_probe_extension/test__sbi_probe_extension.c

```diff
@@ -0,0 +1,43 @@
+#include "test_macros.h"
+
+int test_case(void);
+
+/**
+ * @brief interrupt handler function that runs in S-mode. Not used in this test
+ * 
+ */
+__attribute__((interrupt("supervisor")))
+void s_mode_trap(void) {
+  uint64_t cause = read_csr(scause);
+
+  switch (cause) {
+    default:
+      break;
+  }
+}
+
+/**
+ * @brief Test case example checks for illegal instruction exception
+ * 
+ * @return int 
+ */
+int test_case(void) {
+  //unsigned long input = 0x10ul;
+  struct sbiret get_spec = sbi_probe_extension(EID_10);
```

Comment:
I ran the function on multiple computers, and it ran fine. If its failing please let me know `why` its failing. Attach the spike log and asm log, and mention which `instruction` is causing it to fail.